Source/Core/Core/FifoPlayer/FifoAnalyzer.cpp

```diff
@@ -300,4 +332,15 @@ void LoadCPReg(u32 subCmd, u32 value, CPMemory& cpMem)
     break;
   }
 }
+
+void ClearDrawCache()
+{
+  hash_to_entry.clear();
+  data_to_entry.clear();
+}
+
+FifoDrawCacheEntry* GetDrawCacheEntry(const u8* data)
```

Comment:
This klunkyness is because the 'Analyze' function is used by both the Recorder and the Player.  I was trying not to impact the player if possible.